.github/dependabot.yml

```
@@ -56,3 +56,12 @@ updates:
       # Contains incompatible API changes and needs compatibility work. See:
       # https://github.com/jenkinsci/jenkins/pull/4224
       - dependency-name: "org.jfree:jfreechart"
+  - package-ecosystem: "maven"
+    directory: "/"
+    target-branch: "stable-2.361"
+    labels:
+      - "into-lts"
```

to prevent the "too easy to merge" concern (and IIUC the intent is to flag things as available) should the `needs-justification` label be added here?  (Or perhaps another "here be dragons" style label that flags things as give this more attention than normal.)


```suggestion
      - "into-lts"
      - "needs-justification"
```

---

That makes sense to me, since there ought to be a justification for each and every approved backport. In cases like e.g. #6993 and #7063 such a label is usually not necessary because the justification(s) are usually (though not always) provided in either the developer mailing list or in the ticket(s) where the `lts-candidate` label was added. Since a bot-created backport cannot currently contain a nuanced justification (pending some more advanced AI), making the need for a justification explicit with a label should help to ensure that a justification is always provided and always evaluated prior to the acceptance of a backport request.

---

@jtnord I accepted this suggestion. Are you interested in taking another look at the result?